### PR TITLE
Failure Flash Added

### PR DESCRIPTION
**Pull request by flaredragon**
- On Local Login It prompts on Incorrect Username Or Password
- On OAuth flash is given on Failure of Authentication or Authorization.

### CODE REVIEW

File: passport/strategies/user/strategy-facebook.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ module.exports = new FacebookStrategy({
         }).spread(function(userFacebook, created) {
             //TODO: Check 'created' == true to see if first time user
             if (!userFacebook) {
-                return cb(null, false);
+                return cb(null, false,{message:'Authentication Failed'});
```
Comment (championswimmer) [inline passport/strategies/user/strategy-facebook.js]: `return cb(null, false, {message: 'Authentication Failed'});`

File: passport/strategies/user/strategy-github.js [hunk 2]
```diff
@@ -59,11 +59,11 @@ module.exports = new GithubStrategy({
             }).spread(function(userGithub, created) {
             //TODO: Check created == true for first time
             if (!userGithub) {
-                return cb(null, false);
+                return cb(null, false,{message:'Authentication Failed'});
```
Comment (championswimmer) [inline passport/strategies/user/strategy-github.js]: formatting

File: routers/pages/public.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ router.get('/login', function(req, res, next) {
     res.redirect('/users/me');
   }
   else {
-    res.render('login', {title: "Login | OneAuth"});
+    res.render('login', {title: "Login | OneAuth",error: req.flash('error')});
```
Comment (championswimmer) [inline routers/pages/public.js]: space after comma

**Comment by flaredragon** [comment]
Sorry Sir I've made the changes and will follow the linting pattern to write code from now on.

**Pull request merged**